src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+let elements = [...document.querySelectorAll('.population')];
+
+elements = elements.map((e) => {
+  return +e.textContent.split(',').join('');
+});
+
+let sum = elements.reduce((totall, element) => element + totall, 0);
+let average = Math.ceil(sum / elements.length);
+
+sum = sum.toString();
+average = average.toString();
+
+const formator = (string) => {
+  let result = '';
+  const reversedString = string.split('').reverse();
+
+  for (let i = string.length - 1; i >= 0; i--) {
+    if (i % 3 === 2 && i !== reversedString.length - 1) {
```

Comment:
The logic for adding spaces in the 'formator' function is incorrect. The condition 'i % 3 === 2' does not correctly determine where to place spaces for thousands separators. Consider revising this logic to correctly format the number with commas.